torch_xla/distributed/xla_dist.py

```diff
@@ -115,7 +115,8 @@ def _check_client_mesh_health(
          extra={'clientip': '', 'ordinal': ''})
 
     for cw_hb in self._last_heartbeats.values():
-      max_delay = max(max_delay, now - cw_hb['last_time'])
+      if cw_hb['count'] > 0:
```

Comment:
Until we find better strategy, let's make those timeouts environment configurable instead.

Reply:
Done. Keeping the count=0 case different as that was a bug fix.

Reply:
What does count help here?
At EPOCH 2 count will be > 0 and if there is an issue at 0, there will be even later.

Reply:
Yeah that's true. I was assuming that the data would only be loaded in the first epoch, but you're right it'll push the bug out even further. Disabling this casing and real fix will probably come from resetting the heartbeat at end of rendezvous.